### PR TITLE
small docs update for clarity

### PR DESCRIPTION
**Pull request by makelinux**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ python -m venv venv
 source venv/bin/activate
 pip install -r requirements.txt
 pip install -r requirements-dev.txt
-pip install -e .
+pip install --editable .
```
Comment (nathan-weinberg) [inline README.md:28]: Don't see the need for this change, the others are fine

**Review by nathan-weinberg — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (danmcp) [reply README.md:28]: +1, we are using -e in a lot of places

Comment (makelinux) [reply README.md:28]: Sure, I can remove the change if you insist. Please consider that people are accustomed to using short forms of options, but scripts are indeed more readable with longer, more descriptive options.

Comment (nathan-weinberg) [reply README.md:28]: This is a README file not a script, and hopefully folks working on a Python library would be familiar with this flag

Anyway, please update the commit history, it still shows a commit for the long option, thanks

**Review by khaledsulayman — APPROVED** [review]

**Review by danmcp — APPROVED** [review]

**Pull request merged**